demos/classification_benchmark_demo/cpp_gapi/classification_benchmark_demo_gapi.hpp

```diff
@@ -0,0 +1,77 @@
+// Copyright (C) 2022-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+///////////////////////////////////////////////////////////////////////////////////////////////////
+#pragma once
+#include <iostream>
+
+#include <gflags/gflags.h>
+
+static const char help_message[] = "Print a usage message.";
+static const char image_message[] = "Required. Path to a folder with images or path to an image file.";
+static const char model_message[] = "Required. Path to an .xml file with a trained model.";
+static const char labels_message[] = "Required. Path to .txt file with labels.";
+static const char kernel_package_message[] =
+    "Optional. G-API kernel package type: opencv, fluid (by default opencv is used).";
```

Comment:
We don't use `fluid` at all there, since the graph is pretty straightforward: `in`->`infer`->`topK` there is no operations that could be run with `fluid` backend

Reply:
let's remove it